SocialCareCaseViewerApi.Tests/V1/Factories/EntityFactoryTests.cs

```diff
@@ -460,6 +461,84 @@ public void CreateCaseNoteRequestToCaseNotesDocumentUsesCorrectDateTimeFormatFor
 
             (DateTime.TryParseExact(timestamp, "dd/MM/yyyy HH:mm:ss", CultureInfo.InvariantCulture, DateTimeStyles.None, out DateTime date)).Should().BeTrue();
         }
+
+        [Test]
+        public void CaseSubmissionToCareCaseDataReturnsAssociatedResidentInformation()
+        {
+            var residents = new List<Person> { TestHelpers.CreatePerson(), TestHelpers.CreatePerson() };
+            var request = TestHelpers.CreateListCasesRequest(residents[0].Id);
+            var submission = TestHelpers.CreateCaseSubmission(residents: residents);
+
+            var response = submission.ToCareCaseData(request);
+
+            response.PersonId.Should().Be(residents[0].Id);
+            response.FirstName.Should().Be(residents[0].FirstName);
+            response.LastName.Should().Be(residents[0].LastName);
+        }
+
+        [Test]
+        public void CaseSubmissionToCareCaseDataReturnsOfficeEmailOfFirstWorkerAssociatedWithCaseSubmission()
+        {
+            var residents = new List<Person> { TestHelpers.CreatePerson() };
+            var workers = new List<dbWorker> { TestHelpers.CreateWorker(), TestHelpers.CreateWorker() };
+            var request = TestHelpers.CreateListCasesRequest(residents[0].Id);
+            var submission = TestHelpers.CreateCaseSubmission(workers: workers, residents: residents);
+
+            var response = submission.ToCareCaseData(request);
+
+            response.OfficerEmail.Should().Be(workers[0].Email);
+        }
+
+        [Test]
+        public void CaseSubmissionToCareCaseDataReturnsCaseFormTimeStampFromSubmittedAtIfNotNull()
+        {
+            var residents = new List<Person> { TestHelpers.CreatePerson() };
+            var submittedAt = new DateTime(2021, 07, 20, 14, 40, 30);
+            var request = TestHelpers.CreateListCasesRequest(residents[0].Id);
+            var submission = TestHelpers.CreateCaseSubmission(submittedAt: submittedAt, residents: residents);
+
+            var response = submission.ToCareCaseData(request);
+
+            response.CaseFormTimestamp.Should().Be("2021-07-20");
+        }
+
+        [Test]
+        public void CaseSubmissionToCareCaseDataReturnsCaseFormTimeStampAsDateTimeNowIfSubmittedAtNull()
+        {
+            var residents = new List<Person> { TestHelpers.CreatePerson() };
+            var request = TestHelpers.CreateListCasesRequest(residents[0].Id);
+            var submission = TestHelpers.CreateCaseSubmission(residents: residents, submittedAt: null);
+
+            var response = submission.ToCareCaseData(request);
+
+            response.CaseFormTimestamp.Should().Be(DateTime.Now.ToString("yyyy-MM-dd"));
+        }
```

Comment:
These tests aren't really related to the task at hand (making sure correct createdAt time is returned) but where missing.

They are testing functionality that already existed from the usecase/factory functions.